exercises/06-resource_files.md

```diff
@@ -98,31 +98,31 @@ Login & Invalid Login                                                 | FAIL |
 ==============================================================================
 ```
 
-Why is that? We've it in our `resource.robot`, right?
+Why is that? We have the keywords in our `resource.robot`, right?
```

Comment:
Could/should `.resource` extension be used instead of `.robot` as is [recommended by the user guide](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#taking-resource-files-into-use)?

`.robot` works just as fine and it might be less confusing to participants if left as is, but `.resource` is still recommended.

Reply:
I think it needs to be checked if rflint will actually parse the *.resource. Next bigger thing would be to replace rflint with https://pypi.org/project/robotframework-robocop/

Reply:
Changed the resource file from `resource.robot` to `common.resource` in all occasions. Linter works well with the changes (the filename changed there too).